server/endpoint/githubwebhook/endpoint.go

```diff
@@ -271,7 +270,7 @@ func (e *Endpoint) processDeploymentEvent(ctx context.Context, event *github.Dep
 
 	timeoutSeconds := int64(30)
 	lo := metav1.ListOptions{
-		FieldSelector:  fields.OneTermEqualSelector(api.ObjectNameField, appCRName).String(),
+		FieldSelector:  fields.OneTermEqualSelector("metadata.name", appCRName).String(),
```

Comment:
Don't need to pull the whole pkg just for this const. I will replace this when we introduce [apimachinery v0.22.0](https://github.com/kubernetes/apimachinery/commit/c3f3e5bb4c4b8bd68402247f6ef02a110a491014)